draft-ietf-quic-multipath.md

```diff
@@ -839,27 +839,21 @@ provides more robust measurements.
 
 The transmission of QUIC packets on a regular QUIC connection is regulated
 by the arrival of data from the application and the congestion control
-scheme. QUIC packets can only be sent when the congestion window of
-at least one path is open.
+scheme. QUIC packets can only be sent when the congestion window is open.
 
```

Comment:
Can we say this a little better? ACK frames, for example, are not subject to the congestion control. The proper statement would be:
~~~suggestion
QUIC packets that increase the number of bytes in flight can only be sent when the congestion window is open.
~~~
But that's a bit tautological. One could just drop the whole sentence, because that's exactly what the word "regulated" above means.

Reply:
You're right. Your suggestion above is good (committing now). But yeah, this sentence may not be necessary in the end...